gunicorn/http/message.py

```diff
@@ -312,18 +312,10 @@ def parse_request_line(self, line_bytes):
         self.method = bits[0].upper()
 
         # URI
```

Comment:
Can you delete this line too? New code is descriptive enough so  we don't need it anymore.

Reply:
It is consistent with other comments in this function: [Method](https://github.com/benoitc/gunicorn/blob/cb2f47183153368b70b9c06d14e43a4bf6ac7d92/gunicorn/http/message.py#L309) and [Version](https://github.com/benoitc/gunicorn/blob/cb2f47183153368b70b9c06d14e43a4bf6ac7d92/gunicorn/http/message.py#L333).